current/LAU/items/LAU_RG_01M_2020_3035.json

```diff
@@ -0,0 +1,140 @@
+{
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "LAU_RG_01M_2020_3035",
+  "properties": {
+    "title": "LAU",
+    "description": "A system of local administrative units (LAUs) compatible with NUTS",
+    "table:columns": [
+      {
+        "name": "GISCO_ID",
+        "type": "string",
+        "description": ""
+      },
+      {
+        "name": "CNTR_CODE",
+        "type": "string",
+        "description": ""
+      },
+      {
+        "name": "LAU_ID",
+        "type": "string",
+        "description": ""
+      },
+      {
+        "name": "LAU_NAME",
+        "type": "string",
+        "description": ""
+      },
+      {
+        "name": "POP_2020",
+        "type": "double",
+        "description": ""
+      },
+      {
+        "name": "POP_DENS_2",
+        "type": "double",
+        "description": ""
+      },
+      {
+        "name": "AREA_KM2",
+        "type": "double",
+        "description": ""
+      },
+      {
+        "name": "YEAR",
+        "type": "int64",
+        "description": ""
+      },
+      {
+        "name": "FID",
+        "type": "string",
+        "description": ""
+      },
+      {
+        "name": "geometry",
+        "type": "binary",
+        "description": ""
+      }
+    ],
+    "proj:bbox": [
+      -2824243.6908,
+      -3076198.0823,
+      10026005.2215,
+      5415709.913699999
+    ],
+    "proj:epsg": 3035,
+    "table:row_count": 98613,
+    "created": "2024-07-30T10:05:57.239808Z",
+    "datetime": "2024-01-22T00:00:00Z"
+  },
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          93.1752791130622,
+          -29.857519696177313
+        ],
+        [
+          93.1752791130622,
+          37.855412297701754
+        ],
+        [
+          -81.78251054209933,
+          37.855412297701754
+        ],
+        [
+          -81.78251054209933,
+          -29.857519696177313
+        ],
+        [
+          93.1752791130622,
+          -29.857519696177313
+        ]
+      ]
+    ]
+  },
+  "links": [
+    {
+      "rel": "root",
+      "href": "../../catalog.json",
+      "type": "application/json",
+      "title": "CoCliCo STAC Catalog"
+    },
+    {
+      "rel": "collection",
+      "href": "../collection.json",
+      "type": "application/json",
+      "title": "Local administrative units"
+    },
+    {
+      "rel": "parent",
+      "href": "../collection.json",
+      "type": "application/json",
+      "title": "Local administrative units"
+    }
+  ],
+  "assets": {
+    "data": {
+      "href": "gs://coclico-data-public/coclico/LAU/LAU_RG_01M_2020_3035.parquet",
+      "type": "application/vnd.apache.parquet",
+      "title": "LAU",
+      "description": "A system of local administrative units (LAUs) compatible with NUTS",
+      "roles": [
+        "data"
+      ]
+    }
+  },
+  "bbox": [
```

Comment:
I see a bbox that looks strang in the radiantearth coclico-stac-LAU catalog, could you check?